steamapi.go

```diff
@@ -38,9 +41,10 @@ func (s SteamMethod) Request(data url.Values, v interface{}) error {
 	defer resp.Body.Close()
 
 	if resp.StatusCode != 200 {
-		return errors.New(fmt.Sprintf("Status code %d returned by %v", resp.StatusCode, string(s)))
+		return fmt.Errorf("steamapi %s Status code %d", s, resp.StatusCode)
 	}
 
 	d := json.NewDecoder(resp.Body)
-	return d.Decode(&v)
+	_ = d.Decode(&v)
```

Comment:
I really don't like ignoring JSON errors, that one already brought problems over at go-steam recently; maybe just match for Steam's error message explicitely and then don't JSON decode? Also, wouldn't Steam send some error status code then? That should be handled in the code above.